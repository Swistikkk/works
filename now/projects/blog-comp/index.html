<!DOCTYPE html>
<html lang="en">
<head>
    <link rel="stylesheet" href="css/style.css">
    <meta charset="UTF-8">
    <title>Blog-comp</title>
</head>
<body>
    
    <div class="top-menu">
        <div class="home">
            <a href="#" class="top-home"><img src="img/home.jpg" alt=""></a> 
        </div>
        <div class="top-menu-list">
            <a href="#" class="top-menu-list-item">Новости</a>
            <a href="#" class="top-menu-list-item">Блог</a>
            <a href="#" class="top-menu-list-item">Обучение</a>
            <a href="#" class="top-menu-list-item">Об авторе</a>
            <a href="#" class="top-menu-list-item">Контакты</a>
        </div>
        <div class="menu-icon">
            <a href="#" class="first-menu-icon"><img src="img/search.gif" alt=""></a> 
            <a href="#" class="menu-icon-2"><img src="img/facebook.gif" alt=""></a> 
            <a href="#" class="menu-icon-3"><img src="img/twitter.gif" alt=""></a> 
            <a href="#" class="menu-icon-4"><img src="img/google+.gif" alt=""></a> 
            <a href="#" class="last-menu-icon"><img src="img/profile.png" alt=""></a> 
        </div>
    </div>
    
    <div class="top-header">
        <div class="top-content">
            <h1>Интересная история</h1>
            <p class=top-element>Также все другие известные генераторы Lorem Ipsum 
            <p>используют один и тот же текст, который они просто повторяют,</p>
            <p class="bottom-element">пока не достигнут нужный объём.</p>
            <a href="#" class="top-header-link">Узнать больше</a>
        </div>
    </div>
    
    <div class="container">
       <div class="left-part-of-site">
            <aside class="left-sidebar">
                <div class="section-sidebar">
                    <h3>Технологии</h3>
                    <ul>
                        <li><a href="#" class="sidebar-link-gorizontal">Софт</a></li>
                        <li><a href="#" class="sidebar-link-gorizontal">Гаджеты</a></li>
                        <li><a href="#" class="sidebar-link-gorizontal">Мобильные</a></li>
                    </ul>
                </div>
                <div class="middle-section-bar">
                    <h3>Развлечения</h3>
                    <ul>
                        <li><a href="#" class="sidebar-link-gorizontal">Кино</a></li>
                        <li><a href="#" class="sidebar-link-gorizontal">Игры</a></li>
                        <li><a href="#" class="sidebar-link-gorizontal">Музыка</a></li>
                        <li><a href="#" class="sidebar-link-gorizontal">Спорт</a></li>
                    </ul>
                </div>
                <div class="section-sidebar">
                    <h3>Бизнес</h3>
                    <ul>
                        <li><a href="#" class="sidebar-link-gorizontal">Работа</a></li>
                        <li><a href="#" class="sidebar-link-gorizontal">Маркетинг</a></li>
                        <li><a href="#" class="sidebar-link-gorizontal">Медиа</a></li>
                    </ul>
                </div>
            </aside>
            
            <div class="left-last-posts">
                <h1 class="name-last-posts">Последние записи</h1>
                <div class="last-post">
                    <a href="#" class="image-link-last"><img src="img/left-bar-img-1.jpg" alt=""></a>
                    <p class="name-of-part-sidebar"><a href="">Многие думают, что
                        Lorem Ipsum...</a></p>
                    <time datetime="2015-11-18T18:18:17" class="time-post">3 минуты назад</time>
                </div>

                <div class="last-post">
                    <a href="#" class="image-link-last"><img src="img/left-bar-img-2.jpg" alt=""></a>
                    <p class="name-of-part-sidebar"><a href="">Многие думают, что
                        Lorem Ipsum...</a></p>
                    <time datetime="2015-11-18T18:18:19" class="time-post">7 часов назад</time>
                </div>

                <div class="last-post">
                    <a href="#" class="image-link-last"><img src="img/left-bar-img-3.jpg" alt=""></a>
                    <p class="name-of-part-sidebar"><a href="">Многие думают, что
                        Lorem Ipsum...</a></p>
                    <time datetime="2015-11-18T18:18:18" class="time-post">23 часа назад</time>
                </div>
                
             </div>
             
             <div class="main">
                <div class="form-my">
                    <h2>Задать вопрос</h2>
                    <form>
                        <input type="text" placeholder="Ваше имя">
                        <input type="text" placeholder="Email">
                        <textarea placeholder="Ваш вопрос"></textarea>
                    </form>
                    <button>Отправить</button>
                </div>
             </div>
        
        </div><!--/.left-part-of-site-->
        
        <div class="contant">
            <div class="post">
                <a href="#"><img src="img/post-img-1.jpg" alt=""></a>
                <div class="left-post-link">
                    <a href="#"><img src="img/left-post-img.jpg" alt=""></a>
                </div>
                <div class="post-text">
                    <a href="#" class="zagolovok"><h1 class="name-post">Что такое Loreum Ipsum?</h1></a>
                    <p class="text-post">Давно выяснено, что при оценке дизайна и композиции читаемый текст мешает сосредоточиться. Lorem Ipsum используют потому, что тот обеспечивает более или менее стандартное заполнение шаблона, а также реальное распределение букв и пробелов в абзацах, которое не получается при...<a href="#" class="post-text-link">Читать дальше</a></p>
                    <div class="bottom-post-links">
                        <ul>
                            <li class="reposts"><span>67 Репостов</span></li>
                            <li class="looked"><span>1,257 Просмотров</span></li>
                            <li class="comments"><span>11 комментариев</span></li>
                        </ul>
                    </div>
                </div>
            </div>
            
            <div class="post">
                <a href="#"><img src="img/post-img-2.jpg" alt=""></a>
                <div class="left-post-link">
                    <a href="#"><img src="img/left-post-img.jpg" alt=""></a>
                </div>
                <div class="post-text">
                    <a href="#" class="zagolovok"><h1 class="name-post">Что такое Loreum Ipsum?</h1></a>
                    <p class="text-post">Давно выяснено, что при оценке дизайна и композиции читаемый текст мешает сосредоточиться. Lorem Ipsum используют потому, что тот обеспечивает более или менее стандартное заполнение шаблона, а также реальное распределение букв и пробелов в абзацах, которое не получается при...<a href="#" class="post-text-link">Читать дальше</a></p>
                    <div class="bottom-post-links">
                        <ul>
                            <li class="reposts"><span>27 Репостов</span></li>
                            <li class="looked"><span>3,152 Просмотров</span></li>
                            <li class="comments"><span>21 комментариев</span></li>
                        </ul>
                    </div>
                </div>
            </div>
            
            <div class="post">
                <a href="#"><img src="img/post-img-3.jpg" alt=""></a>
                <div class="left-post-link">
                    <a href="#"><img src="img/left-post-img.jpg" alt=""></a>
                </div>
                <div class="post-text">
                    <a href="#" class="zagolovok"><h1 class="name-post">Что такое Loreum Ipsum?</h1></a>
                    <p class="text-post">Давно выяснено, что при оценке дизайна и композиции читаемый текст мешает сосредоточиться. Lorem Ipsum используют потому, что тот обеспечивает более или менее стандартное заполнение шаблона, а также реальное распределение букв и пробелов в абзацах, которое не получается при...<a href="#" class="post-text-link">Читать дальше</a></p>
                    <div class="bottom-post-links">
                        <ul>
                            <li class="reposts"><span>37 Репостов</span></li>
                            <li class="looked"><span>752 Просмотров</span></li>
                            <li class="comments"><span>3 комментариев</span></li>
                        </ul>
                    </div>
                </div>
            </div>
        </div><!--/.contant-->
    </div><!--/.container-->
    <footer class="bottom-footer">
        <div class="left-part-of-footer">
            <a href="http://coolwebsite.com" class="copirites">www.coolwebsite.com</a><span> &copy; 2015</span>
        </div>
        <div class="right-part-of-footer">
            <a href="#">Главная</a>
            <a href="#">Новости</a>
            <a href="#">Блог</a>
            <a href="#">Обучение</a>
            <a href="#">Об авторе</a>
            <a href="#">Контакты</a>
        </div>
    </footer>
</body>
</html>