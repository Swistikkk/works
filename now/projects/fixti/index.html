<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Fixti.me</title>
    <link rel="stylesheet" href="css/style.css">
    <link href='https://fonts.googleapis.com/css?family=Roboto:400,700,300' rel='stylesheet' type='text/css'>
    <script src="http://code.jquery.com/jquery-2.2.0.min.js"></script>
    <link rel="stylesheet" type="text/css" href="css/slick.css"/>
<script type="text/javascript" src="http://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
<script type="text/javascript" src="libs/slick.min.js"></script>
</head>
<body>
   <div class="obertka mk-slider mk-edge-slider   js-el" id="mk-edge-slider-569d92ebac158">
    <div data-type="background" data-speed="5" class="header">
        
        <div class="top-navigation">
            <div class="logo">
                <a href="#">
                    <img src="img/logo.png" alt="Fixti.me">
                </a>
            </div>
            
            <nav class="top-menu">
                <a href="#" class="inline-bl">Возможности</a>
                <a href="#" class="inline-bl_2">Цены</a>
                <a href="#" class="inline-bl_3">Академия подписки</a>
                <a href="#" class="active">Начать бесплатно</a>
                <a href="#" class="inline-bl_4">Войти</a>
            </nav>
        </div>
        
        <div class="header-contant">
<p class="white-bg">Subscription e-commerce platform</p>
            <h1>Всё, что нужно, чтобы начать</h1>
            <span class="header-contant-offer">продавать по подписке</span>
       
       <p class="funct">Полный функционал 14 дней бесплатно</p>
       <form class="top-form">
           <input id="email-empty" type="email" name="email" value="E-mail">
           <button class="btn pri">Создать магазин</button>
           <span>Регистрация меньше, чем за минуту</span>
       </form>
       
       
       
       
       
       
       <section id="section07" class="demo">
	<a href="#your_block"><span></span><span></span><span></span></a>
</section>
       
       
       
       
       
       
        </div>  
    </div>
</div>
       
       <div  id="your_block" ></div>
       
        <div class="steps">
            <div class="step-item">
                <span>1/</span>
                <img src="img/first_step.jpg" alt="Магазин">
                <p>Создайте свой магазин по подписке</p>    
            </div>

            <div class="step-item">
                <span>2/</span>
                <img src="img/second_step.png" alt="База">
                <p>Набирайте базу постоянных покупателей</p>    
            </div>    
            
            <div class="step-item right">
                <span>3/</span>
                <img src="img/third_step.png" alt="Управляй">
                <p>Управляйте своим бизнесом</p>    
            </div> 
        </div><!--/steps-->




        <div class="bg-fon">
            <div class="formats">
                <div class="formats-contant">
                    <h3>Два формата продаж по подписке</h3>  
                    <div class="formats-item">
                        <p class="unline"><span>Супермаркет товаров и услуг по подписке </span><br>Subscription supermarket</p>
                        <p class="grey">Покупатель сам собирает свою корзину,<br>также как в магазине и сам настраивает<br>периодичность доставки</p>
                    </div>
                    <div class="formats-item">
                        <p class="unline"><span>Подписка на коробочное предложение</span><br>Subscription box</p>
                        <p class="grey">Покупатель выбирает вашу“коробку”,<br>собранный набор продуктов с<br>заданным Вами графиком доставки</p>
                    </div>    
                </div>
                <div class="formats-sliders">
                    <div class="computer">
                        <img src="img/computer.png" alt="">
                        <div class="wrapper-for-slider">
                         <div class="my-slider">
                            <div><img src="img/test-photo.jpg" alt=""></div>
                               <div><img src="img/test-photo.jpg" alt=""></div>

                        </div>   
                        </div>
                    </div>
                </div>
            </div>
        </div>
        
        <div class="platform">
            
             <h3>Subscription e-commerce platform</h3>
             <p>Fixti.me - это комплексная система электронной коммерции по подписке номер 1 в России
<br>Полный комплекс инструментов для рекуррентных продаж</p>
           <nav class="hover_a">
               

                   <span href="#" class="block_a active_a_trio"  id="block_1">Современный интернет-магазин</span>
                   <span href="#" class="block_a" id="block_2">Реферальная программа для покупателей</span>
                   <span href="#" class="block_a" id="block_3">Прием онлайн оплаты</span>
                   <span href="#" class="block_a" id="block_4">Организация доставки через курьерские службы</span>
                   <span href="#" class="block_a" id="block_5">Аналитика продаж</span>

           </nav>   
            
           <div class="platform-text block_id"  block="block_1" style="display: block;">
               <div class="platform-text-img">
                   <img src="img/site.png" alt="Современный интернет магазин">
               </div>
               <div class="platfo-p"><p>Полный контроль над товарами</p><p>Корзина онлайн-покупок, полностью хранящаяся на нашем сервере</p><p>Безопасность на высочайшем уровне</p><p>SEO-оптимизированные страницы товара</p><p>Удобное управление заказами как для продавца, так и для покупателя</p><p>Напоминание о покупках покупателя</p>
               </div>
           </div>
           <div class="platform-text block_id"  block="block_2">
               <div class="platform-text-img">
                   <h3>В разработке</h3>
               </div>
               <div class="platfo-p"> </div>
           </div>
          <div class="platform-text block_id"  block="block_3">
               <div class="platform-text-img">
                   <h3>В разработке</h3>
               </div>
               <div class="platfo-p"> </div>
           </div>
           <div class="platform-text block_id"  block="block_4">
               <div class="platform-text-img">
                   <h3>В разработке</h3>
               </div>
               <div class="platfo-p"> </div>
           </div>
           <div class="platform-text block_id"  block="block_5">
               <div class="platform-text-img">
                   <h3>В разработке</h3>
               </div>
               <div class="platfo-p"> </div>
           </div>
        </div>
        
        <div class="priority">
            
           <div class="priority-item">
               <div class="priority-item-img"><img src="img/priority_1.jpg" alt=""></div>
               <div class="priority-text">
                    <p>Какие возможности открывает fixti.me покупателю</p>
                   <a href="priority-button">Читать</a>   
                   </div>
                   <div class="hover_effect"></div>
           </div>
           <div class="priority-item">
               <div class="priority-item-img"><img src="img/priority_2.jpg" alt=""></div>
               <div class="priority-text">
                    <p>Условия оплаты, доставки и возврата товаров</p>
                   <a href="priority-button">Читать</a>   
                   </div>
                   <div class="hover_effect"></div>
           </div>
           <div class="priority-item">
               <div class="priority-item-img"><img src="img/priority_3.jpg" alt=""></div>
               <div class="priority-text">
                    <p>Зачем и как начать продавать по подписке</p>
                   <a href="priority-button">Читать</a>   
                   </div>
                   <div class="hover_effect"></div>
           </div>
           <div class="priority-item">
               <div class="priority-item-img"><img src="img/priority_4.jpg" alt=""></div>
               <div class="priority-text">
                    <p>Академия подписки.
Всё, что нужно об этом знать</p>
                   <a href="priority-button">Читать</a>   
                   </div>
                   <div class="hover_effect"></div>
           </div>  
        </div>
        
        <div class="footer">
            <div class="footer-wrapper">
                <div class="footer-left">
                    <div class="for-buy">
                        <p>Покупателю</p>
                        <a href="#">Личный кабинет покупателя</a>
                        <a href="#">Каталог товаров</a>
                        <a href="#">Список магазинов</a>
                        <a href="#">Каталог коробочных подписок</a>
                        <a href="#">Условия доставки и возврата</a>
                        <a href="#">Пользовательские соглашения</a>
                        <a href="#">Политика конфидециальности</a>
                        <a href="#">Возможности</a>
                        <a href="#">Помощь</a>
                        <a href="#">Контакты</a>
                    </div>
                </div>
                </div>
                <div class="footer-right">
    <div class="follow">
        <p>Подписаться на fixti.me</p>
        <div class="facebook">
            <a href="#"><img src="img/facebook.png" alt="facebook"></a>
        </div>
        <div class="twitter">
            <a href="#"><img src="img/twitter.png" alt="twitter"></a>
        </div>
        <div class="vk">
            <a href="#"><img src="img/vk.png" alt="vk"></a>
        </div>
    </div>
    <div class="language">
        <p>Активный язык</p>
            <select class="active">
                <option value="Русский">Русский</option>
                <option value="English">English</option>
            </select>
        <span>2015-2016 &copy; fixti.me</span>
    </div>
    </div>
                <div class="for-sale">
                        <div class="for-sale-step"><p>Продавцу</p>
                        <a href="#">Личный кабинет продаца</a>
                        <a href="#">О проекте</a>
                        <a href="#">Как начать продавать</a>
                        <a href="#">Тарифы</a>
                        <a href="#">Академия подписки</a></div>
                    <a href="#" class="active footer-link">Открыть магазин</a>    
                    </div>
            </div>
    <!--JavaScript-->
    <script src="libs/wp-emoji-release.min.js"></script>
    <script src="libs/jquery.smoothscroll.js"></script>
    <script src="js/main.js"></script>
</body>
</html>