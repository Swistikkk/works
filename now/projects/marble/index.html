<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Marble</title>
    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,700|Roboto+Condensed:400,700|Roboto' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    
    <div class="header-bg">
        
        <div class="header-logo full-size">
            <a href="#" class="header-logo__a">Marble</a>
            
            <nav>
                <a href="#" class="active">Home</a>
                <a href="#">About</a>
                <a href="#">Journal</a>
                <a href="#">Service</a>
                <a href="#">Features</a>
                <a href="#">Contact</a>
            </nav>
        </div>
        
    </div>
    
    <div class="header-img">
       <h1>We are a digital & branding agency based in London.</h1>
       <h3>We love to turn great ideas into beautiful products.</h3>
       <div><a href="#">see portfolio</a></div>
    </div>
    
    
     <div class="full-size"> 
       <div class="b-advantages">
            <div class="b-advantages__img"></div>
            <h2>Pellentesque</h2>
            <p>Nulla vitae  libero, a pharetra augue. Integer posuere erat a ante venenatis condimentum velit dapibus. </p>
     </div>
     
     <div class="b-advantages">
            <div class="b-advantages__img"></div>
            <h2>Consectetur</h2>
            <p>Vestibulum id ligula porta  euismod semper. Aenean lacinia bibendum nulla sed consectetur dui.</p>
     </div>
     
     <div class="b-advantages">
            <div class="b-advantages__img"></div>
            <h2>Tristiquet</h2>
            <p>Fusce dapibus, cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo risus.</p>
     </div>
     
     <div class="b-advantages">
            <div class="b-advantages__img"></div>
            <h2>Fermentum</h2>
            <p>Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum sagittis lacus mollis.</p>
     </div>
     <div class="border-center"></div>
    </div>
    
    <div class="b-latest-works">
        <h2>our latest works</h2>
        <div class="work work__first">
            <img src="img/img_1.jpg" alt="">
            <h3>Nobis Business Card</h3>
            <span>Business Card, Graphic</span>
            <p>Sed posuere consectetur est at lobortis. Porta sagittis lacus vel augue laoreet rutrum.</p>
        </div>
        
        <div class="work">
            <img src="img/img_2.jpg" alt="">
            <h3>New Fun Project</h3>
            <span>Web Design, Application</span>
            <p>Nulla vitae elit libero, a pharetra augue. Donec sed odio dui. Vestibulum id ligula porta felis.</p>
        </div>
        
        <div class="work">
            <img src="img/img_3.jpg" alt="">
            <h3>Passionaries Branding Cover</h3>
            <span>Branding, Graphic Design</span>
            <p>Curabitur blandit tempus porttitor. Donec id elit non mi porta gravida at eget metus.</p>
        </div>
    </div>
    
    <div class="quote">
        <div>
            <img src="img/quote.png" alt="">
            <p>Cras mattis consectetur purus sit amet fermentum. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus erat porttitor. </p>
        </div>
    </div>
    
    <div class="footer-b">
        <div class="full-size">
           <div class="footer-item">
                <h3>Get In Touch</h3>
                <p>Fusce dapibus, tellus commodo, tortor mauris condimentum utellus fermentum, porta sem malesuada magna.</p>
                <ul class="b-list b-list__contacts">
                    <li>Moonshine Street No: 14/05 
    Light City, Jupiter</li>
                    <li>0247 541 65 87</li>
                    <li>support@longwave.com</li>
                </ul>
           </div>
           
           <div class="footer-item">
                <h3>Twitter</h3>
                <ul class="b-list b-list__twitter">
                    <li>Aenean lacinia bibendum nulla sed leo posuere erat a ante venenatis dapibus posuere velit aliquet.</li>
                    <li>Donec ullamcorper metus auctor fringilla. Nullam quis risus eget.</li>
                    <li>Vestibulum id ligula porta  euismod semper. Maecenas faucibus mollis.</li>
                </ul>
           </div>
           
           <div class="footer-item">
                <h3>Popular Posts</h3>
                <ul class="b-list b-list__posts">
                    <li><a href="#">Vivamus sagittis lacus vel augue laoreet rutrum dolor auctor.</a><time datetime="2015-12-11">14 Nov, 2012</time></li>
                    <li><a href="#">Scelerisque nisl consectetur et.</a><time datetime="2015-12-11">22 May, 2012</time></li>
                    <li><a href="#">Pellentesque ornare sem lacinia quam venenatis vestibulum.</a><time datetime="2015-12-11">15 July, 2012</time></li>
                </ul>
           </div>
           
           <div class="footer-item">
                <h3>About us</h3>
                    <ul class="b-list b-list__about">
                    <li>Vestibulum id ligula porta felis euismod semper. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. .</li>
                    <li>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus curabitur blandit tempus.</li>
                </ul>
    </div>
        </div>
</div>
<div class="bottom-bg-img">
          <div class="copyright">
           <p>&copy; 2013 Marble. All rights reserved. Theme by elemis.</p>
       </div>
</div>
</body>
</html>