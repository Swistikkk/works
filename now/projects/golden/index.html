<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Golden</title>
    <link rel="stylesheet" href="css/style.css">
    <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400italic,400|Montserrat:400,700' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" href="css/animate.css">   
</head>
<body>
    <div class="b-header">
            <div class="b-header__menu">
                <a href="#" class="b-header__link"><img src="img/header-logo.png" alt=""></a>
                <nav>
                    <a href="#">Home</a>
                    <a href="#">Services</a>
                    <a href="#">Portfolio</a>
                    <a href="#">About</a>
                    <a href="#">Contact</a>
                </nav>
            </div>
            
            <div class="top-banner wow rubberBand">
                <h3>Welcome To Our Studio!</h3>
                <h2>Itâ€™s nice to Meet you</h2>
                <a href="#" class="btn">tell me more</a>
            </div>
    </div>
    
    <div class="b-services">
        
        <div class="heading_of_b wow rotateInUpRight" data-wow-delay="0.8s">
            <h2>Services</h2>
            <p>Proin iaculis purus consequat sem cure. </p>
        </div>
        
        <div class="b-services-item wow bounceIn" date-wow-duration="1s" data-wow-delay="1s">
            <div class="b-services-item__img"></div>
            <h4>E-Commerce</h4>
            <p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
        </div>
        
        <div class="b-services-item wow bounceIn" date-wow-duration="1s" data-wow-delay="1.2s">
            <div class="b-services-item__img"></div>
            <h4>Responsive Web</h4>
            <p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
        </div>
        
        <div class="b-services-item wow bounceIn" date-wow-duration="1s" data-wow-delay="1.4s">
            <div class="b-services-item__img"></div>
            <h4>Web Security</h4>
            <p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
        </div>
    </div>
    
    <div class="portfolio">
         
        <div class="heading-of-p wow rotateInUpRight">
             <h2>Our portfolio</h2>
             <p>Proin iaculis purus consequat sem cure. </p>
        </div>
        
        <div class="centrovka">
            <div class="portfolio-item wow rollIn" data-wow-delay="0.8s">
                <a href="#"><img src="img/work_1.jpg" alt=""></a>
                <h4>Ebony & Ivory</h4>
                <p>Branding</p>
            </div>
            
            <div class="portfolio-item wow rollIn" data-wow-delay="1s">
                <a href="#"><img src="img/work_2.jpg" alt=""></a>
                <h4>Smart Stationary</h4>
                <p>Print Design</p>
            </div>
            
            <div class="portfolio-item wow rollIn" data-wow-delay="1.2s">
                <a href="#"><img src="img/work_3.jpg" alt=""></a>
                <h4>Clever Poster</h4>
                <p>Print Design</p>
            </div>
            
            <div class="portfolio-item wow rollIn" data-wow-delay="1.4s">
                <a href="#"><img src="img/work_4.jpg" alt=""></a>
                <h4>Vinyl Record</h4>
                <p>Product Mock-Up</p>
            </div>
            
            <div class="portfolio-item wow rollIn" data-wow-delay="1.6s">
                <a href="#"><img src="img/work_5.jpg" alt=""></a>
                <h4>Treehouse Template</h4>
                <p>Web Design</p>
            </div>
            
            <div class="portfolio-item wow rollIn" data-wow-delay="1.8s">
                <a href="#"><img src="img/work_6.jpg" alt=""></a>
                <h4>Burned Logo</h4>
                <p>Branding</p>
            </div>
        </div>
    
    </div>
    
    <div class="b-about">
        
        <div class="heading-of-a wow rotateInUpRight">
            <h2>About Us</h2>
            <p>Proin iaculis purus consequat sem cure. </p>
        </div> 
        
        <div class="about-item">
            <div class="about-contant-left wow rotateInDownLeft"  data-wow-delay="1.6s">
                <h3 class="big">July 2010</h3>
                <h3>Our Humble Beginnings</h3>
                <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
            </div>    
            <div class="about-contant-right"></div>
            <div class="about-img"><img src="img/about_1.jpg" alt=""></div>
        </div>
        
        <div class="poloska"></div>
        
        <div class="about-item">
            <div class="about-contant-left"></div>    
            <div class="about-contant-right  wow rotateInDownRight"  data-wow-delay="1.6s">
                <h3 class="big">January 2011</h3>
                <h3>Facing Startup Battles</h3>
                <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
            </div>
            <div class="about-img"><img src="img/about_2.jpg" alt=""></div>
        </div>
        
        <div class="poloska"></div>
        
        <div class="about-item">
            <div class="about-contant-left wow rotateInDownLeft"  data-wow-delay="1.6s">
                    <h3 class="big">December 2012</h3>
                    <h3>Enter The Dark Days</h3>
                    <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
            </div>    
            <div class="about-contant-right"></div>
            <div class="about-img"><img src="img/about_3.jpg" alt=""></div>
        </div>
        
        <div class="poloska"></div>
        
        <div class="about-item">
            <div class="about-contant-left"></div>    
            <div class="about-contant-right wow rotateInDownRight"  data-wow-delay="1.6s">
                <h3 class="big">February 2014</h3>
                <h3>Our Triumph</h3>
                <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
            </div>
            <div class="about-img"><img src="img/about_4.jpg" alt=""></div>
        </div>
        
        <div class="poloska"></div>
        
        <div class="about-item">
            <div class="about-contant-left"></div>    
            <div class="about-contant-right"></div>
            <div class="about-continues">
                <a href="#">our<br>story<br> continues<br>...</a>
            </div>
        </div>
    </div>
    
    <div class="team">
        <div class="header-of-t wow rotateInUpRight">
            <h2>Our Amazing Team</h2>
             <p>Proin iaculis purus consequat sem cure. </p>
        </div>
        
        
        <div class="team-center">
            <div class="team-item wow wobble" data-wow-delay="0.8s">
                <div class="team-photo">
                    <img src="img/team_1.jpg" alt="">
                </div>
                <div class="team-contant">
                    <h4>Kimberly Thompson</h4>
                    <p>Marketer</p>
                    <a href="#" class="twitter"></a>
                    <a href="#" class="facebook"></a>
                    <a href="#" class="painted"></a>
                </div>
            </div>

            <div class="team-item  wow wobble" data-wow-delay="1s">
                <div class="team-photo">
                    <img src="img/team_2.jpg" alt="">
                </div>
                <div class="team-contant">
                    <h4>Rico Massimo</h4>
                    <p>Coder</p>
                    <a href="#" class="twitter"></a>
                    <a href="#" class="facebook"></a>
                    <a href="#" class="google"></a>
                </div>
            </div>

            <div class="team-item  wow wobble" data-wow-delay="1.2s">
                <div class="team-photo">
                    <img src="img/team_3.jpg" alt="">
                </div>
                <div class="team-contant">
                    <h4>Uku Mason</h4>
                    <p>Graphic Designer</p>
                    <a href="#" class="facebook"></a>
                    <a href="#" class="painted"></a>
                </div>
            </div>
        </div>
        
        <div class="quoter">
            <p>Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit  
aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
        </div>
    </div>
    
    <div class="bottom-logos">
        <a href="#"><img src="img/bottom_1.jpg" alt=""></a>
        <a href="#"><img src="img/bottom_2.jpg" alt=""></a>
        <a href="#"><img src="img/bottom_3.jpg" alt=""></a>
        <a href="#"><img src="img/bottom_4.jpg" alt=""></a>
    </div>
    
    <div class="contacts-us">
        <div class="contant-us-c wow rotateInUpRight">
           <div class="map"></div>
            
            <div class="header-of-contacts">
                <h2>Contacts Us</h2>
            <p>Proin iaculis purus consequat sem cure. </p>
            </div>
            
            <div class="contacts-form">
                <form class="left">
                    <input type="text" name="first-name" class="width-of-i" placeholder="your Name *"><br>
                    <input type="email" name="email" class="width-of-i" placeholder="your e-mail *"><br>
                    <input type="text" name="subject"  class="width-of-i" placeholder="subject"><br>
                </form>
                
                <form class="right">
                    <textarea class="contant-messege" placeholder="Your message *"></textarea>
                </form>
            </div>
            <a href="#" class="btn-center">Send Massege</a>
        </div>
    </div>
    
    <div class="copyrights">
        <p>&copy; Copyright 2014 FreebiesXpress.com</p>
        <a href="" class="twitter_b"></a>
        <a href="" class="facebook_b"></a>
        <a href="" class="painted_b"></a>
        <a href="" class="google_b"></a>
    </div>

<!--JavaScript-->
<script src="js/wow.min.js"></script>
<script src="js/main.js"></script>

</body>
</html>